docker/android-slave/.s2i/bin/run

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
```

Comment:
@luigizuccarelli I meant to document this here - Just add additional comment here